.editorconfig

```diff
@@ -6,9 +6,13 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-trim_trailing_whitespaces = true
 indent_style = tab
-quote_type = single
```

Comment:
Why did you remove this?

Reply:
It's not officially supported: https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#ideas-for-domain-specific-properties

Reply:
OK, reverted it, because **maybe** some IDEs use it (I'm not aware of it *g*)